custom-login/package.json

```diff
@@ -13,16 +13,15 @@
     "postinstall": "npm run patch-semantic-ui-css --prefix ../"
   },
   "dependencies": {
-    "@okta/okta-auth-js": "^7.8.1",
-    "@okta/okta-signin-widget": "^7.24.2",
+    "@okta/okta-auth-js": "~7.9.0",
+    "@okta/okta-signin-widget": "~7.26.1",
```

Comment:
Why the change from compatible to approximate version range? IIUC this will prevent update to `@okta/okta-auth-js@7.10.0` and `@okta/okta-signin-widget@7.27.0` respectively

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies
https://docs.npmjs.com/cli/v6/using-npm/semver#tilde-ranges-123-12-1